flower-shop/flower_shop.py

```diff
@@ -0,0 +1,67 @@
+flowers_list = []
```

Comment:
Explain why you have decided to use list as a main flower data storage? 

Imagine the situation when we have bouquets with different flowers, and with different flowers quantity

Like this: Bouquet Alpha (Roses: 5, Lilys: 8, SomeFlower: 3)
And the same way Shop could have Bouquet Alpha : 5, Another Bouquet: 8

Wouldn't it be better to use dictionary in this situation?